global_hub/global_hub_import_hub_default.adoc

```diff
@@ -5,10 +5,12 @@ To import an existing hub cluster as a managed hub cluster, complete the followi
 
 . Disable the cluster self-management in the existing {acm} hub cluster by setting the `disableHubSelfManagement` setting to `true` in the `multiclusterhub` custom resource. This setting disables the automatic importing of the hub cluster as a managed cluster.
 
-. Import the managed hub cluster by completing the steps in link:../clusters/cluster_lifecycle/import_intro.adoc#import-intro[Cluster import introduction].
+. Import the managed hub cluster by completing the steps in link:../clusters/cluster_lifecycle/import_intro.adoc#import-intro[Cluster import introduction]. You must set the label `global-hub.open-cluster-management.io/deploy-mode=default` to the managedcluster when you import it.
 
 . After the managed hub cluster is imported, check the {global-hub} agent status to ensure that the agent is running in the managed hub cluster by running the following command:
 +
 ----
-oc get managedclusteraddon multicluster-global-hub-controller -n $<managed_hub_cluster_name>
-----
\ No newline at end of file
+oc get managedclusteraddon multicluster-global-hub-controller -n <managed_hub_cluster_name>
+----
+
+Note: If you try to upgrade the Globalhub from 1.4.x, you must mannually add the label `global-hub.open-cluster-management.io/deploy-mode=default` to all managed hub clusters.
```

Comment:
what's the impact if the label is not added before the upgrade?

Reply:
> what's the impact if the label is not added before the upgrade?

The globalhub agent will be unsintall from the managedhub, user could also add the label after upgrade.